zerver/management/commands/soft_activate_deactivate_users.py

```diff
@@ -16,6 +18,17 @@
 )
 from zerver.lib.management import ZulipBaseCommand
 
+log_format = "%(asctime)s: %(message)s"
+logging.basicConfig(format=log_format)
+
+formatter = logging.Formatter(log_format)
+file_handler = logging.FileHandler(settings.SOFT_DEACTIVATION_LOG_PATH)
+file_handler.setFormatter(formatter)
+
+logger = logging.getLogger('')
+logger.setLevel(logging.INFO)
+logger.addHandler(file_handler)
```

Comment:
I think you could just do `from zerver.lib.soft_deactivation import logger` rather than duplicating the code; it'd also save a bit of initialization time.

Also, now that we have this pattern N times in the codebase, maybe it's worth having a library helper function of the form `get_zulip_logger(settings.SOFT...)` that does all this?  Seems silly to have block copied in like 5 files.